projects/igniteui-angular/src/lib/services/overlay/overlay.spec.ts

```diff
@@ -504,6 +504,42 @@ describe('igxOverlay', () => {
             expect(BaseFitPositionStrategy.prototype.position).toHaveBeenCalledTimes(3);
         });
 
+        it('Should properly call setOffset method', fakeAsync(() => {
+            const fix = TestBed.createComponent(WidthTestOverlayComponent);
+            const overlayInstance = fix.componentInstance.overlay;
+            const id = fix.componentInstance.overlay.attach(SimpleRefComponent);
+
+            overlayInstance.show(id);
+
+            fix.detectChanges();
+            tick();
+
+            overlayInstance.setOffset(id, 40, 40);
+            const overlayContent = document.getElementsByClassName(CLASS_OVERLAY_CONTENT_MODAL)[0];
+            const component = document.getElementsByClassName('simpleRef')[0];
+            const contentRectOverlay = overlayContent.getBoundingClientRect();
+            const componentRectOverlay = component.getBoundingClientRect();
+
+            expect(contentRectOverlay.top).toEqual(componentRectOverlay.top);
+            expect(contentRectOverlay.left).toEqual(componentRectOverlay.left);
+
+            // Set the offset again and verify it is changed correctly
+            overlayInstance.setOffset(id, -10, 20);
+            fix.detectChanges();
+            tick();
+            const contentRectOverlayNew = overlayContent.getBoundingClientRect();
+            const componentRectOverlayNew = component.getBoundingClientRect();
+
+            expect(contentRectOverlayNew.top).toEqual(componentRectOverlayNew.top);
+            expect(contentRectOverlayNew.left).toEqual(componentRectOverlayNew.left);
+
+            expect(contentRectOverlayNew.top).not.toEqual(contentRectOverlay.top);
+            expect(contentRectOverlayNew.left).not.toEqual(contentRectOverlay.left);
+
+            expect(componentRectOverlayNew.top).not.toEqual(componentRectOverlay.top);
+            expect(componentRectOverlayNew.left).not.toEqual(componentRectOverlay.left);
+        }));
```

Comment:
same here, add `expect` for `overlayContent` element if it has correct `transform` style